acsoo/pylintcmd.py

```diff
@@ -93,7 +93,10 @@ def do_pylintcmd(load_plugins, rcfile, module, expected, pylint_options):
         + list(module)
     )
     log_cmd(["pylint"] + cmd, level=logging.INFO)
-    lint_res = pylint.lint.Run(cmd[:], exit=False)
+    try:
+        lint_res = pylint.lint.Run(cmd[:], do_exit=False)  # pylint2
+    except TypeError:
+        lint_res = pylint.lint.Run(cmd[:], exit=False)  # pylint1
```

Comment:
@ThomasBinsfeld it looks like this changed occured with pylint >= 2.

So better test the installed version:
```
import pkg_resources
pkg_resources.get_distribution("pylint") >= pkg_resources.parse_version("2.0")
```

Reply:
@sbidoul Done